### ISSUE TITLE
output not complete

### ISSUE THREAD

**Issue by mountrcg** [marionbarker/OmniLoopMessageParser#2]
I try to analyze eros pod logs from FreeAPS X. Normally I used to get

- determBasalDF220219_out.csv
- Unknown_20220219_plot.png

However with the last master I get an error
```
/Users/RCG/Projects/OmniLoopMessageParser/util/pod.py:193: FutureWarning: Dropping of nuisance columns in DataFrame reductions (with 'numeric_only=None') is deprecated; in a future version this will raise TypeError.  Select only valid columns before calling the reduction.
  logInfoDict['totBolus'] = bolusState.sum()['reqBolus']
      Missing podDict, check report for ## OmnipodPumpManager section
```
and the 2 files

- logDFCmb_out.csv
- podState_Unknown_20220628_1.csv
get generated.

I use:
```
# run the most recent file and append to master csv
from main import main
from fileio.get_file_list import get_file_list, getFileDict
from fileio.getAnalysisIO import getAnalysisIO

pathOption = 1  # use standard file location
loopType = 'FX'  # FreeAPS X
vFlag = 4  # Verbose output - all files to Output location
macFlag = 1  # 0 = use Drobo; 1 use Mac hard drive

folderPath, outFlag = getAnalysisIO(pathOption, loopType, vFlag, macFlag)
fileDateList = get_file_list(folderPath)

# create fileDict
fileDict = getFileDict(folderPath, fileDateList[-1][0], loopType)

# nominal verbose output
main(fileDict, outFlag, vFlag)
```
Anything I can do to fix the missing podDict?

**Comment by mountrcg** [comment]
the only thing in the log that is changed since then are the OpenAPS entries in the logfile, they are more numerous and contain more information, all changes from determine_basal.js (oref console output)

**Comment by marionbarker** [comment]
I'm sorry but I'm so focused on keeping things working with Loop-dev that I've been ignoring your issue.  I know I had to make a bunch of changes as the log and log_prev files changed to support other FAX folks. But I don't have the bandwidth right now. I only added this capability when I was using FAX (because there was no Apple HealthKit connection at the time and I wanted to track my TDD). I have not used FAX in a year. 

If you figure it out and want to make a pull request back to this repo I'll review and accept it. Look at the commit history to see other changes made for FAX to see if that offers a clue.

Pretty sure podDict is only populated from Loop Report files. There is no equivalent reporting in FAX. But haven't really looked at your issue in detail.
